db/migrate/20200412174115_add_status_to_order.rb

```diff
@@ -0,0 +1,5 @@
+class AddStatusToOrder < ActiveRecord::Migration[5.1]
+  def change
+    add_column :orders, :status, :string, :default => "Pending"
```

Comment:
Should the default be Pending? 

I know these are the four that an order can have
'pending' means a user has placed items in a cart and "checked out" to create an order, merchants may or may not have fulfilled any items yet
'packaged' means all merchants have fulfilled their items for the order, and has been packaged and ready to ship
'shipped' means an admin has 'shipped' a package and can no longer be cancelled by a user
'cancelled' - only 'pending' and 'packaged' orders can be cancelled